MigrationScript/Migration.ps1

```diff
@@ -1,18 +1,18 @@
 ﻿param(
 #[Parameter(Mandatory=$true)]
-[string]$subscriptionId = "<subscription_id>",
+[string]$subscriptionId = "49d64d54-e966-4c46-a868-1999802b762c",
```

Comment:
I am a bit concerned about this IDS...Why hard code them ?

Reply:
yeah will remove them, added by mistake